enzyme/test/Enzyme/ReverseMode/blas/cblas_ddot_inactive_second.ll

```diff
@@ -0,0 +1,85 @@
+;RUN: %opt < %s %loadEnzyme -enzyme -mem2reg -instsimplify -simplifycfg -S | FileCheck %s
+
+;#include <cblas.h>
+;
+;extern double __enzyme_autodiff(double*, double*, double*);
+;
+;double g(double *restrict n) {
+;    double m[3] = {1, 2, 3};
+;	 double x = cblas_ddot(3, m, 1, n, 1);
+;    double y = x*x;
```

Comment:
Can you add a test where the stride is an inactive argument to g?

Reply:
Would this be an appropriate example? 

```
float g(float *restrict m, float *restrict n) {
  int stride = 3;
  float x = cblas_sdot(3, m, 2, n, stride);
  float y = x * x;
  return y;
}

int main() {
  float m[6] = {1, 2, 3, 101, 102, 103};
  float m1[6] = {0, 0, 0, 0, 0, 0};
  float n[9] = {4, 5, 6, 104, 105, 106, 7, 8, 9};
  float n1[9] = {0, 0, 0, 0, 0, 0, 0, 0, 0};
  __enzyme_autodiff((void*)g, m, m1, n, n1);
}
```